lib/server.js

```diff
@@ -351,7 +351,7 @@ class PropertyHandler extends BaseHandler {
       try {
         thing.setProperty(propertyName, req.body[propertyName]);
       } catch (e) {
-        res.status(403).end();
+        res.status(400).end();
```

Comment:
Ideally this could pick up `e.message` or some other info

Reply:
I've kept all of the error codes simple for now until we actually define what error responses should look like in the spec.